### PR TITLE
chore: use width 100 for iframe

### PR DESCRIPTION
**Pull request by BandanaKM**
### Overview

Small change to make the width 100% for the iframe version so it takes up the full width and length of the page. Prior to this, we were getting a fixed width. 

### PR Checklist

- [x] I have run this code to verify it works

### CODE REVIEW

**Review by jakemhiller — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/codebytes@0.6.3-alpha.e74dd5.0<br/></pre>

File: packages/codebytes/src/codeByteEditor.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { trackUserImpression } from './libs/eventTracking';
 import { CodeByteEditorProps } from './types';
 
 const editorStates = states({
-  isIFrame: { height: '100vh' },
+  isIFrame: { height: '100vh', width: '100%' },
```
Comment (borisonr) [inline packages/codebytes/src/codeByteEditor.tsx:16]: any reason to use vh for height and not vw for width?

Comment (BandanaKM) [reply packages/codebytes/src/codeByteEditor.tsx:16]: i tried vw for width, but it's adding a weird scroll, and so opted to go with `100%` width

Comment (borisonr) [reply packages/codebytes/src/codeByteEditor.tsx:16]: gotcha all good

**Review by borisonr — APPROVED** [review]

**Pull request merged**